<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="aprendizado">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Aprendizado :</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>84</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="hebb">
       <property name="text">
        <string>Hebb</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="percep">
       <property name="text">
        <string>Perceptron</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="treinamento">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Treinamento</string>
    </property>
   </widget>
   <widget class="QPushButton" name="treinar_but">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>321</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Treinar</string>
    </property>
   </widget>
   <widget class="QLabel" name="pesos_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>380</y>
      <width>41</width>
      <height>99</height>
     </rect>
    </property>
    <property name="text">
     <string>Pesos : </string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>51</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="peso1">
       <property name="text">
        <string>w1 = 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="peso2">
       <property name="text">
        <string>w2 = 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pesobaias">
       <property name="text">
        <string>wb = 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="rodando">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>90</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Rodando :</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>100</y>
      <width>160</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="dados">
       <property name="text">
        <string>Dados : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="dado1">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="singleStep">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="dado2">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="singleStep">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="testar">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Testar</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultado">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>270</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Resultado : </string>
    </property>
   </widget>
   <widget class="QLabel" name="inputs_t">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>88</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Input :</string>
    </property>
   </widget>
   <widget class="QLabel" name="Input_value">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputs">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Outputs:</string>
    </property>
   </widget>
   <widget class="QLabel" name="outpu_values">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>369</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="epoca">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>410</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Epocas = </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>786</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>